### ISSUE TITLE
scss error in angular environement

### ISSUE THREAD

**Issue by lilworks** [contentco/quill-emoji#47]
I use ngx-quill. When I try to import quill-emoji I got this error
```
ERROR in ./node_modules/quill-emoji/src/scss/quill-emoji.scss
Module Error (from ./node_modules/postcss-loader/src/index.js):
(Emitted value instead of an instance of Error) CssSyntaxError: /Users/lil-works1/Webdisk/www/npm/ngx-quill-angular-cli/node_modules/quill-emoji/src/scss/core/_emoji.scss:13:22: Can't resolve '../../sheet_apple_64.png' in '/Users/lil-works1/Webdisk/www/npm/ngx-quill-angular-cli/node_modules/quill-emoji/src/scss'
```
How can I solve this?

**Comment by sumitvekariya** [comment]
Here is the pull request solving this error.

https://github.com/contentco/quill-emoji/pull/53

**Comment by cwhite-connectfirst** [comment]
If you're using webpack, you need to add "resolve-url-loader" to the loaders you use for your scss styles.  This is what I have in my webpack config, in `module.rules`:

```
{
        test: /\.s?css$/,
        use: [
            {
                loader: 'style-loader',
            },
            { loader: 'css-loader' },
            { loader: 'resolve-url-loader' }, // used to import emojis properly in quill-emoji
            {
                loader: 'sass-loader',
                options:
                    {   // options are required by resolve-url-loader (see https://github.com/bholloway/resolve-url-loader/blob/master/packages/resolve-url-loader/README.md)
                        sourceMap: true,
                        sourceMapContents: false
                    }
            },
        ]
    };
```

**Comment by CFT-Chris** [comment]
Confirmed that the absence of ``'resolve-url-loader'`` in webpack.config for scss will evoke an error message with path being unable to be resolved, similar to what this issue and #56 talk about.

@lilworks please check your webpack.config and verify if cwhite-connectfirst's suggestion solves your issue.

**Comment by CFT-Chris** [comment]
#68 should fix this. In the duplicate issue #56, I mentioned:

> any projects with quill-emoji as a dependency shouldn't be importing the raw scss files and should be consuming the ``dist/quill-emoji.css`` file.

> AFAIK, building the quill-emoji project from this repo works with the current relative path ([webpack.config](https://github.com/contentco/quill-emoji/blob/v0.1.6/webpack.config.js#L31) correctly has ``resolve-url-loader``) and shouldn't be changed.
